src/DIRAC/ConfigurationSystem/Client/Utilities.py

```diff
@@ -462,7 +462,6 @@ def getElasticDBParameters(fullname):
         # No host name found, try at the common place
         result = gConfig.getOption("/Systems/NoSQLDatabases/Host")
         if not result["OK"]:
-            gLogger.warn("Failed to get the configuration parameter: Host. Using localhost")
```

Comment:
I am thinking that actually removing them altogether is a bit too aggressive. Maybe we keep them, but adding a boolean flag in the call of the method?

Reply:
Do you just want a boolean flag in the function that we set manually if we need? Or do you also want a mechanism to call it with/without the flag from a service/agent? (e.g. a CS parameter)